src/components/pages/contact/leaflet/leaflet.tsx

```diff
@@ -1,20 +1,25 @@
+import * as L from 'leaflet';
+import { LatLngExpression } from 'leaflet';
+import { GestureHandling } from 'leaflet-gesture-handling';
+import 'leaflet-gesture-handling/dist/leaflet-gesture-handling.css';
```

Comment:
I guess we don't need those imports if we put them into React Helmet below with:

```
 <link
          rel="stylesheet"
          href="//unpkg.com/leaflet-gesture-handling/dist/leaflet-gesture-handling.min.css"
          type="text/css"
        />
```

It's better to use the React Helmet import, instead of `import`

Reply:
👍